README.md

```diff
@@ -1,12 +1,74 @@
-# About
-GitHub Action to execute tests by Testing Farm and update Pull Request status.
+# testing-farm-as-github-action
 
-The Action uses ubuntu-20.04
+Testing Farm as GitHub Action is a GitHub Action to execute tests by Testing Farm and update Pull Request status.
 
-## Prerequisities
+This action only runs on `Linux`. Before a calling this GitHub Action, you must first clone your project by e.g. 
+GitHub Action [Checkout V2](https://github.com/actions/checkout).
 
-Before scheduling tests on Testing Farm, developer is responsible for checkouting repo, eg. by actions/checkout@v2.
-In your organization secrets you should have stored API key to testing farm.
+The Action uses ubuntu-20.04.
 
 Important binaries, like curl, jq, which are used for getting scheduled jobs from Testing Farm are already installed
-by this GitHub Action.
\ No newline at end of file
+by this GitHub Action.
+ 
+In your organization secrets you should have stored API key to testing farm.
+
+## Action Inputs
+
+| Input Name      | Description             | Default |
+|-----------------|-------------------------| ------- |
+| api_key         | A testing farm API key. | **Must be provided** |  
+| tmt_repository | An url to tmt repository | **Must be provided** |
+| test_fmf_plan | A fmf plan which will be selected. | All plans |
```

Comment:
What means "All plans"? It is not clear from this table.

Reply:
It means all plans in fmf plan.